scripts/pre_push_hook.py

```diff
@@ -272,11 +272,23 @@ def start_linter(files):
     return task.returncode
 
 
-def start_python_script(scriptname):
-    """Runs the 'start.py' script and returns the returncode of the task."""
+def run_script_and_get_returncode(script_name):
+    """Runs script and returns the returncode of the task.
+    Args:
```

Comment:
Please _always_ follow existing code conventions. Separate sections in docstring by one line. Use the correct python typeinfo for the args and returns values. Use 4-space indentation.

Reply:
Done.

Reply:
Still missing a typeinfo for script_name.

Reply:
Done.